sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverClient.java

```diff
@@ -28,6 +28,8 @@
  */
 @ServiceClient(builder = ServiceBusClientBuilder.class)
 public class ServiceBusReceiverClient implements AutoCloseable {
+    private static final Duration DEFAULT_RECEIVE_WAIT_TIME = Duration.ofMinutes(1);
```

Comment:
This was discussed during review that instead of using tryTimeout, we should be using some default value 1 minute or less.

Reply:
The default AmqpRetryOptions has the timeout already set. This isn't necessary. Or if they pass their own default tryTimeout.